augmentedreality/Common/source/ArcGISArViewInterface.cpp

```diff
@@ -355,16 +355,9 @@ void ArcGISArViewInterface::setPlaneColor(const QColor& planeColor)
 /*!
   \internal
  */
-void ArcGISArViewInterface::geometryChanged(const QRectF& newGeometry, const QRectF& oldGeometry)
+void ArcGISArViewInterface::geometryChange(const QRectF& newGeometry, const QRectF& oldGeometry)
 {
-  // enable detection of orientation changes.
-  if (window() && window()->screen())
-  {
-    window()->screen()->setOrientationUpdateMask(Qt::LandscapeOrientation	| Qt::PortraitOrientation |
-                                                 Qt::InvertedLandscapeOrientation | Qt::InvertedPortraitOrientation);
-  }
```

Comment:
Is there any equivalent orientation logic for Qt 6? @GuillaumeBelz any thoughts here?

Reply:
`setOrientationUpdateMask` was used to limit the signal `orientationChanged` to a list of selected orientations. By default, the mask was 0, then no signal was emitted. We set the mask to receive the signal when the orientation was changed.

Now, the mask is removed and signal is always emitted.